yeast-markdown-parser/src/__tests__/tests/MarkdownParser.test.ts

```diff
@@ -394,24 +384,13 @@ test('MarkdownParser using all inline plugins', () => {
 	// Parse
 	const documentText = fs.readFileSync(path.join(__dirname, '../resources/everythinginline.md'), 'utf8');
 	const ast = parser.parse(documentText);
+	// debugAST('everythinginline', ast);
 
 	// Check document
-	checkAstStructureForDefaultDocument(ast, 1);
```

Comment:
I removed these coded checks in favor of direct/full AST comparison, but made sure all the existing tests were passing before making that change.